src/system/SystemConfig.h

```diff
@@ -320,6 +320,18 @@
 #define CHIP_SYSTEM_CONFIG_PACKETBUFFER_POOL_SIZE 15
 #endif /* CHIP_SYSTEM_CONFIG_PACKETBUFFER_POOL_SIZE */
 
+/**
+ *  @def CHIP_SYSTEM_CONFIG_PACKETBUFFER_LWIP_PBUF_TYPE
+ *
+ *  @brief
+ *      LwIP @pbuf_type for System::PacketBuffer allocations.
+ *
+ *      Note that this does not affect allocations by LwIP itself, e.g. the normal receive path.
+ */
+#ifndef CHIP_SYSTEM_CONFIG_PACKETBUFFER_LWIP_PBUF_TYPE
+#define CHIP_SYSTEM_CONFIG_PACKETBUFFER_LWIP_PBUF_TYPE PBUF_RAM
```

Comment:
The comments in https://github.com/project-chip/connectedhomeip/issues/13843 don't make it sound like this is the right thing?

Reply:
Agreed. I do not think this is the true root cause, and may mask the true root cause.

Reply:
Hence not tagging this with ‘Fixes’. I think we still want this option, but should `PBUF_POOL` remain the default for now?

Reply:
Yes, PBUF_POOL has the remain the default since it has been default for 2 years without others raising the need to change it.

It very well SHOULD be changed to be configurable, but I think the correct way to do it is to discuss the alternatives, rather than this big late change.